pages/components/Buttons/button.css

```diff
@@ -0,0 +1,117 @@
+@import url(/css/main.css);
+
+.buttons-preview-container {
+    display: flex;
+    flex-wrap: wrap;
+    height: fit-content;
+    border: 1px solid var(--secondary-faded-color);
+    border-radius: 8px;
+    padding: 1rem;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.btn{
+    cursor: pointer;
+    border-radius: 10px;
+    border: none;
+    background-color: transparent;
+    font-size: 1rem;
+    padding: 0.8rem 1rem;
+}
+
+
```

Comment:
s

Reply:
Hey Shivang, thanks for the suggestion, I added the feature.